app/src/test/java/com/example/screenshotpractice/ShowkaseScreenshotTest.kt

```diff
@@ -18,16 +21,23 @@ import org.robolectric.annotation.GraphicsMode
 class ShowkaseScreenshotTest(
     private val showkaseBrowserComponent: ShowkaseBrowserComponent,
 ) {
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<DummyActivity>()
```

Comment:
Lottie already support Prevent infinite Lottie animations. but it only works with createAndroidComposeRule.

Reply:
Lottie-android 1987